.ci/test-spelling-unknown-words.sh

```diff
@@ -72,6 +73,8 @@ diff_output=$(diff -U1 "$whitelist_path" "$run_output" |grep -v "$spellchecker"
 if [ -z "$diff_output" ]; then
   echo "No new words and misspellings found."
   rm $dict
+  rm $word_splitter
+  rm $run_output
```

Comment:
https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=12962&view=logs&j=6a833393-c551-5000-0916-5000fd3b86e7&t=bdbbd129-77f0-5d73-ce7d-db58d34c8358&l=36
Clean up has to be done on an exit 0.
None of the others really need to change because they only occur when there will be a failure anyways.